app/models/form/lettings/pages/scheme.rb

```diff
@@ -6,6 +6,7 @@ def initialize(_id, hsh, subsection)
         "needstype" => 2,
       },
     ]
+    @header = "Scheme"
```

Comment:
are we intenionally not showing the question number anymore?

Reply:
Oh, I think we still do! Just not for 2022, which was the case already